priv/repo/migrations/20190409070736_add_perm_tables.exs

```diff
@@ -0,0 +1,41 @@
+defmodule Radiator.Repo.Migrations.AddPermTables do
+  use Ecto.Migration
+
+  def up do
+    create table(:networks_perm, primary_key: false) do
+      add :user_id, references("auth_users", on_delete: :delete_all), primary_key: true
+      add :network_id, references("networks", on_delete: :delete_all), primary_key: true
+      add :permission, :string, size: 16, null: false
+
+      timestamps()
+    end
+
+    create table(:podcasts_perm, primary_key: false) do
+      add :user_id, references("auth_users", on_delete: :delete_all), primary_key: true
+      add :podcast_id, references("podcasts", on_delete: :delete_all), primary_key: true
+      add :permission, :string, size: 16, null: false
+
+      timestamps()
+    end
+
+    create table(:episodes_perm, primary_key: false) do
+      add :user_id, references("auth_users", on_delete: :delete_all), primary_key: true
+      add :episode_id, references("episodes", on_delete: :delete_all), primary_key: true
+      add :permission, :string, size: 16, null: false
+
+      timestamps()
+    end
+
+    flush()
+
+    Radiator.Repo.insert(Radiator.Auth.User.reserved_user(:public))
```

Comment:
My initial idea was to use the rights of the reserved_user(:public) to define the public published queries to make sure only stuff is public that is. Don't know if that is a good idea anymore, but it is still around for now until I have the read side of the permissions going.

Reply:
Interesting. A downside I see is that one _could_ give this user more rights and suddenly everyone has more rights -- opposed to hardcoding public access.

Reply:
Well, in my head it would also give another context and frontend, but use mainly the same stuff underneath. Maybe we can get best of both worlds here without accidentally giving the public more access. But then again it might be more annoying to exclude those users in all other interfaces. Let's see.